common/src/main/scala/org/apache/celeborn/common/protocol/message/ControlMessages.scala

```diff
@@ -910,6 +936,9 @@ object ControlMessages extends Logging {
         if (!pbHeartbeatFromWorker.getActiveShuffleKeysList.isEmpty) {
           activeShuffleKeys.addAll(pbHeartbeatFromWorker.getActiveShuffleKeysList)
         }
+
+        val workerStatus = PbSerDeUtils.fromPbWorkerStatus(pbHeartbeatFromWorker.getWorkerStatus)
```

Comment:
If the old worker and new master, `pbHeartbeatFromWorker.getWorkerStatus` will be null?

Reply:
> If the old worker and new master, `pbHeartbeatFromWorker.getWorkerStatus` will be null?

For pb, It would get the default instance of PbWorkerStatus